src/AutorestSwift/main.swift

```diff
@@ -31,7 +31,8 @@ guard let documentsUrl = FileManager.default.urls(for: .documentDirectory, in: .
     fatalError("Unabled to locate Documents directory.")
 }
 
-let sourceUrl = documentsUrl.appendingPathComponent("code-model-v4-2.yaml")
+let sourceUrl = documentsUrl.appendingPathComponent("communication-chat-code-model-v4-2.yaml")
+// let sourceUrl = documentsUrl.appendingPathComponent("code-model-v4-2.yaml")
```

Comment:
I'd recommend reverting this change and just using the hard coded file `code-model-v4-2.yaml` for the time being.

Reply:
sorry. this is not meant for checkin